### PR TITLE
Implement EPR export

### PR DESCRIPTION
**Pull request by cintamani**
From: https://eaflood.atlassian.net/browse/RUBY-737

This implements the EPR export as per old system.

### CODE REVIEW

File: app/presenters/reports/registration_epr_presenter.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module Reports
+  class RegistrationEprPresenter < ::WasteCarriersEngine::BasePresenter
```
Comment (Cruikshanks) [inline app/presenters/reports/registration_epr_presenter.rb]: ❤️ Awesome solution to dealing with WCR's data 'inconsistencies'!

File: config/initializers/time_formats.rb [hunk 2]
```diff
@@ -6,3 +6,5 @@
 Time::DATE_FORMATS[:weekday_day_month_year_at_time] = "%A %e %B %Y at %l:%M%P"
 # For example: 2 December 2019
 Time::DATE_FORMATS[:day_month_year] = "%d %B %Y"
+# For example: 2019-12-02
+Time::DATE_FORMATS[:reports_year_month_day] = "%Y-%m-%d"
```
Comment (Cruikshanks) [inline config/initializers/time_formats.rb]: I'd just suggest a slight tweak on the name. I don't think we need to refer to where this is currently used. Plus it means it's inconsistent with the other formats. So I'm suggesting just stick with a name that describes the format itself.

```ruby
Time::DATE_FORMATS[:year_month_day] = "%Y-%m-%d"
# or
Time::DATE_FORMATS[:year_month_day_hyphens] = "%Y-%m-%d"
```

Comment (cintamani) [reply config/initializers/time_formats.rb]:
```suggestion
Time::DATE_FORMATS[:year_month_day_hyphens] = "%Y-%m-%d"
```

File: app/presenters/reports/registration_epr_presenter.rb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    delegate :uprn, :house_number, :address_line_1, :address_line_2, :address_line_3, :address_line_4,
+             :town_city, :postcode, :country, :easting, :northing,
+             to: :company_address, prefix: true
+
+    def metadata_date_activated
+      metaData&.date_activated&.to_formatted_s(:reports_year_month_day)
```
Comment (cintamani) [inline app/presenters/reports/registration_epr_presenter.rb]:
```suggestion
      metaData&.date_activated&.to_formatted_s(:year_month_day_hyphens)
```

File: app/presenters/reports/registration_epr_presenter.rb [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    def registration_type
+      return "carrier_broker_dealer" if lower_tier?
+
+      super
+    end
+
+    def expires_on
+      return if lower_tier?
+
+      super&.to_formatted_s(:reports_year_month_day)
```
Comment (cintamani) [inline app/presenters/reports/registration_epr_presenter.rb]:
```suggestion
      super&.to_formatted_s(:year_month_day_hyphens)
```

File: config/initializers/time_formats.rb (hunk 2 above)
Comment (cintamani) [reply config/initializers/time_formats.rb]:
```suggestion
Time::DATE_FORMATS[:year_month_day_hyphens] = "%Y-%m-%d"
```

File: Gemfile [hunk 5]
```diff
@@ -53,7 +53,7 @@ gem "defra_ruby_aws", "~> 0.2.0"
 # Use the waste carriers engine for the user journey
 gem "waste_carriers_engine",
     git: "https://github.com/DEFRA/waste-carriers-engine",
-    branch: "master"
+    branch: "737-epr-export"
```
Comment (cintamani) [inline Gemfile]:
```suggestion
    branch: "master"
```

**Pull request merged**